container-conf/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+build_image_base=radiasoft/beamsim
+build_is_public=1
+
+build_as_root() {
+    umask 022
+    build_yum config-manager \
+        --add-repo \
+        https://download.docker.com/linux/fedora/docker-ce.repo
+    build_yum install fedora-workstation-repositories
+    build_yum config-manager --set-enabled google-chrome
+    build_yum install google-chrome-stable docker-ce-cli
```

Comment:
While we can install the docker-ce-cli, we can't necessarily test with it. We need a dockerd to talk to.

Did something fail to cause the need for docker-ce-cli or was this just copied from sirepo/container-conf/build.sh?

Reply:
This was copied from sirepo/container-conf/build.sh. Should I install chrome a different way?

Reply:
chrome install is fine. Tt was the docker-ce-cli that's problematic for ci testing. I'm not sure how we'd do that in a github action so I don't know there's any point in adding it to this particular container. This would simplify the container build, e.g. https://download.docker.com/linux/fedora/docker-ce.repo could go away, too.

Reply:
4fb9d93522f934c96ba6a6972f7fcd36692f5fa2